tests/test_namespaces.py

```diff
@@ -101,7 +111,7 @@ def namespace_disallowlist(tmp_path):
         dedent(
             """
             ---
-            disallowed
+            - disallowed*
```

Comment:
I think it would be good to still test that non-wildcards work:
```suggestion
            - disallowed
            - disallowed_*
```